web/static/js/components/stage_help.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import ReactDOM from "react-dom"
+
+import { connect } from "react-redux"
+import { bindActionCreators } from "redux"
+import { actions } from "../redux/retro"
+
+import * as AppPropTypes from "../prop_types"
+import stageConfig from "../configs/stage_configs"
+
+export const StageHelp = props => {
+  const handleClick = () => {
+    const { retro, actions: { showStageHelp } } = props
+    showStageHelp(retro)
+  }
+
+  const showIcon = () => {
+    const { retro: { stage } } = props
+    return (stageConfig[stage] && stageConfig[stage].help)
+  }
+
+  if (showIcon()) {
+    return ReactDOM.createPortal(
+      <div className="portal">
+        <i
+          title="Access Stage Information"
+          className="question circle icon"
+          onClick={handleClick}
+          onKeyPress={handleClick}
+        />
+      </div>,
+      document.getElementById("stage-help-icon")
+    )
+  }
+
+  return null
+}
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
@codyduval the `<RemoteRetro />` component also has bound `actions`, so i think you can kill the `connect` call outright and pass the `actions` object down

Reply:
peep https://github.com/stride-nyc/remote_retro/blob/master/web/static/js/components/remote_retro.jsx#L100-L102

Reply:
Hmm - I've never passed an action creator already bound to dispatch.  Lemme see if that works.

Reply:
Yup works.